### PR TITLE
add UITableView extensions, StoryboardType protocol and StoryboardReference

### PR DESCRIPTION
**Pull request by peuf0u**

### CODE REVIEW

**Review by zvonicek — APPROVED** [review]

File: StoryboardType.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//
+//  StoryboardType.swift
+//  FuntastyKit
+//
+//  Created by Patrik Potoček on 20.12.16.
+//  Copyright © 2016 The Funtasty. All rights reserved.
```
Comment (kafejo) [inline StoryboardType.swift]: Wrong copyright guys 😁

Comment (zvonicek) [reply StoryboardType.swift]: You mean wrong *Created by*? Copyright is IMHO correct, I guess 😉

Comment (kafejo) [reply StoryboardType.swift]: Exactly. https://github.com/kafejo/Storyboard-reference 🙃

Comment (zvonicek) [reply StoryboardType.swift]: Sure, but this code is originally from Maarked where it is with your name and Funtasty copyright. Thus wrong *Created by*.

Comment (peuf0u) [reply StoryboardType.swift]: Sorry, I didn't mean that.

**Pull request merged**